source/include/mqtt_agent.h

```diff
@@ -233,6 +233,77 @@ typedef struct CommandInfo
  * callback MUST remain in scope throughout the period that the agent task is running.
  *
  * @return Appropriate status code from MQTT_Init().
+ *
+ * <b>Example</b>
+ * @code{c}
+ *
+ * // Function for obtaining a timestamp.
+ * uint32_t getTimeStampMs();
+ * // Callback function for receiving packets.
+ * void incomingPublishCallback( struct MQTTAgentContext * pMqttAgentContext,
```

Comment:
I think this should just use the typedef
```suggestion
 * void incomingPublishCallback( MQTTAgentContext_t * pMqttAgentContext,
```